src/main/java/com/github/k0kubun/builder/query/graphql/builder/ObjectBuilderImpl.java

```diff
@@ -9,53 +9,54 @@
 import java.util.List;
 import java.util.Map;
 
-public class GraphQLObjectBuilder
+class ObjectBuilderImpl
```

Comment:
This is package private so that it isn't visible outside of the query builder. This is done so that users of this project don't create this class directly because they are supposed to use just GraphQLQueryBuilder